### ISSUE TITLE
UX improvement: Allow only one voice message to play at a time

### ISSUE THREAD

**Issue by pranavmishra90** [vector-im/element-web#17439]
#### Is your suggestion related to a problem? Please describe.
When using the Element Labs "voice message" feature, it is possible to have multiple voice messages playing back at the same time. Additionally, when while one voice message is playing, if the user goes to a different room, then returns to the original room with the voice message, it is not possible to stop playback. 

#### Describe the solution you'd like.
Starting the playback on any voice message stops playback on any previously playing voice message.

#### Describe alternatives you've considered.
Adds voice message playback to a queue. 

#### Additional context
![image](https://user-images.githubusercontent.com/62562712/119218478-1aafd300-baa6-11eb-8484-5d86b0f1e627.png)
Multiple playback at once 

**After switching to another room, and returning to this one, all three files continue to play, despite looking like they are in a resting state**
![image](https://user-images.githubusercontent.com/62562712/119218502-387d3800-baa6-11eb-8563-4bb75d1f2171.png)

**Comment by gaelledel** [comment]
@turt2live 
- [ ] Only allow one message to play at a time
- [ ] Put in place a queuing system. If the user plays a new recording, it stops the current recording playback at the time that it was stopped e.g: 1:21 and plays the new one. Once the new one is finished playing, the other one plays again from where it stopped at 1:21.
- [ ] If the user navigates out and back to the room, the recordings should simply stop playing and return to their default state.

**Comment by ShadowJonathan** [comment]
Re: last one, I'd argue that a user press "play" on a voice message and then expects it to keep playing in the background, to multi-task and "play incoming voicemail", of sorts.

Though that would need a new design and behaviour for it (and so a new issue), maybe it could re-use the same modals as VoIP? And then show a mini-player in the corner of the screen, thus place it orthogonally alongside other "voice" or "multimedia connectivity"-related things.

**Comment by gaelledel** [comment]
Based on research, not for typical users of voice messages. For this iteration of the MVP we will not include this functionality. Rest assured that we will be monitoring general feedback after the first release and if significant enough, we will consider it for the next iteration.

**Comment by ShadowJonathan** [comment]
Alright, thanks, I was going off of personal assumptions, but that sounds right as well. Thanks for correcting! 👍

**Comment by turt2live** [comment]
Conclusion from an internal call: We'll do the above minus the queuing, which has been broken out to https://github.com/vector-im/element-web/issues/17961

**Comment by turt2live** [comment]
@ShadowJonathan I would encourage your concern to be a new issue ftr
